server/src/auth/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const jwtConstants = {
+    secret: 'someSecretKey',
+};
```

Comment:
This is fine for now but please add a TODO to ensure that we load the secret key from environment.

Alternatively, load the variable from environment in this PR